### PR TITLE
bip174: link to 'compact size' definition, document version 0 handling of > 0xFD

### PR DESCRIPTION
**Pull request by Sjors**

### CODE REVIEW

**Comment by Sjors** [comment]
cc @achow101

**Comment by apoelstra** [comment]
concept ack

File: bip-0174.mediawiki [hunk 1]
```diff
@@ -729,6 +729,8 @@ If an updater is updating a PSBT and needs to add a field that is only available
 New fields should first be proposed on the bitcoin-dev mailing list.
 If a field requires significatn description as to its usage, it should be accompanied by a separate BIP.
 The field must be added to the field listing tables in the Specification section.
+New fields with type 0xFD or higher must be added to PSBT version 2 or newer,
```
Comment (achow101) [inline bip-0174.mediawiki:732]: I don't think this restriction is necessary. Old parsers will regard such fields as unknown anyways, and with the length before the type, no data will be missed.

Comment (Sjors) [reply bip-0174.mediawiki]: I rephrased it to explain why it's safe.

**Comment by achow101** [comment]
ACK 6366f8ebcbab1bc5e59325bf8a00ed96065bd671

**Pull request merged**